lib/getsongslist.js

```diff
@@ -0,0 +1,28 @@
+const request = require('request');
+
+module.exports = function getSongsList(keyArray) {
+  const keysJoined = keyArray.join('+');
```

Comment:
@girdhariag , there is a chance of the keyArray being something like `['sicilian pizza', 'pepperoni']`(returned by Google's Vision API). When you do a `keyArray.join( )`, keysJoined will be `sicilian%20pizza+pepperoni` which doesn't work with our songs API. A quick workaround would be doing something like `keyArray.join('+').replace(' ', '+')`. If you have a better solution for this, that would be awesome as well.

I will merge this PR once this is fixed 👍